Formula/glib.rb

```diff
@@ -1,12 +1,23 @@
 class Glib < Formula
-  include Language::Python::Shebang
+  # Sync everything to Formula/libglib.rb
+  def self.libglib?
+    name.end_with?("Libglib")
+  end
```

Comment:
When would this be true?

Reply:
This would be true in `libglib.rb` and always be false in `glib.rb`.

Reply:
Then why keep it at all?

Reply:
It can make `libglib.rb` identical to `glib.rb` except bottle URLs.

Reply:
But it'll also make it much harder to read and therefore maintain. Why not just make a formula for libglib?